pkg/internal/checkup/executor/executor.go

```diff
@@ -115,7 +115,13 @@ func (e Executor) Execute(ctx context.Context, vmiUnderTestName, trafficGenVMINa
 		return status.Results{}, fmt.Errorf("failed to run traffic from traffic generator VMI \"%s/%s\" side: %w",
 			e.namespace, trafficGenVMIName, err)
 	}
-	time.Sleep(e.testDuration)
+
+	var err error
+	trafficGeneratorMaxDropRate, err := trexClient.MonitorDropRates(ctx, trafficGenVMIName)
```

Comment:
Could you please elaborate why do we need to monitor the drop rates?

Reply:
As we talked offline - there is no practical use of trafficGeneratorMaxDropRate, as it is only sampling and is not a measurable quantity we can use to determine 100% if there were any drops.
Having said that, it is much better than just waiting on a sleep.